routes/api/auth.js

```diff
@@ -164,7 +165,38 @@ module.exports = {
             Middleware.Auth.parseAccountConfirmationToken,
             Middleware.Auth.validateConfirmationToken,
             Controllers.Auth.confirmAccount
-        )
+        );
+
+        /**
+         * @api {get} /auth/rolebindings/:id retrieve rolebindings for a user given by their user id :id
+         * @apiName getRoleBindings
+         * @apiGroup Auth
+         * @apiVersion 0.0.8
+         * 
+         * @apiParam (param) {ObjectId} id MongoId of an account
+         * 
+         * @apiSuccess {string} message Success message
+         * @apiSuccess {object} data Rolebindings object
+         * @apiSuccessExample {object} Success-Response: 
+         *      {
+                    "message": "Successfully retrieved role bindings", 
+                    "data": {RoleBindingsObject}
```

Comment:
Can you try expanding this one a small roleBindings object? You can make up the attribute values, but it would be pretty helpful